Formula/qt5.rb

```diff
@@ -199,6 +199,13 @@ def install
               /\n# pkgconfig\n(PKG_CONFIG_(SYSROOT_DIR|LIBDIR) = .*\n){2}\n/,
               "\n"
 
+    # Make `HOMEBREW_PREFIX/lib/qt5/plugins` an additional plug-in search path
+    # for Qt Designer to support formulae that provide Qt Designer plug-ins.
+    system "/usr/libexec/PlistBuddy",
+           "-c", "Add :LSEnvironment:QT_PLUGIN_PATH string \"#{HOMEBREW_PREFIX}/lib/qt5/plugins\"",
+           "#{bin}/Designer.app/Contents/Info.plist"
+    touch "#{bin}/Designer.app" # re-register edited bundle with Launch Services
```

Comment:
That's fine, it just feels very weird to be doing manual modification of a upstream generated plist. That should be a `configure` option. Would be good to try it with them rather than have effectively a patch we have to carry forever.